src/helpers/mailer.js

```diff
@@ -0,0 +1,36 @@
+import { config } from 'dotenv';
+import sgMail from '@sendgrid/mail';
+
```

Comment:
@timi-codes I think Lawrence is also implementing email sending and you may want to rub mind with him on this. I feel we could have a generic method that does that so anyone implementing email sending could use that .

Reply:
Thank you @kechyy . We will work on this